descriptors/ModuleDescriptor-template.json

```diff
@@ -677,6 +682,11 @@
         "kb-ebsco.kb-credentials.access-types.item.delete"
       ]
     },
+    {
```

Comment:
will this permission be included in some of the "all" permission set?

Reply:
Thanks! Fixed